notebooks/official/ray_on_vertex_ai/spark_on_ray_on_vertex_ai.ipynb

```diff
@@ -0,0 +1,1353 @@
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "4c8784686d72"
+      },
+      "source": [
+        "### Monitor the job logs\n",
+        "\n",
+        "The job logs can also be viewed via Ray on Vertex AI OSS Dashboard in a web browser."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "08c360a961dc"
+      },
+      "outputs": [],
+      "source": [
+        "client.get_job_logs(job_id)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "a97150123bdc"
+      },
+      "source": [
+        "## Reading Cloud Storage files from Spark application\n",
+        "\n",
+        "The following section shows two different techniques for reading Cloud Storage files from Spark applications running on Ray on Vertex AI."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "180671e0a120"
+      },
+      "source": [
+        "### Create a Cloud Storage bucket"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "b6aa1c2aabae"
+      },
+      "outputs": [],
+      "source": [
+        "BUCKET_URI = f\"gs://your-bucket-name-{PROJECT_ID}-unique\"  # @param {type:\"string\"}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "2b4cec7c939b"
+      },
+      "source": [
+        "**If your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "e837b03f84f3"
+      },
+      "outputs": [],
+      "source": [
+        "! gsutil mb -l {LOCATION} -p {PROJECT_ID} {BUCKET_URI}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8fe5968f0d8a"
+      },
+      "source": [
+        "#### Copy [CSV file](https://www.datavis.ca/gallery/guerry/guerry.csv) to GCS bucket."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "56143d1ce75b"
+      },
+      "outputs": [],
+      "source": [
+        "! curl https://www.datavis.ca/gallery/guerry/guerry.csv | gsutil cp - {BUCKET_URI}/guerry.csv"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "87467aad45e4"
+      },
+      "source": [
+        "### Use the Google Cloud Storage  Connector\n",
+        "\n",
+        "The [Google Cloud Connector for Hadoop](https://github.com/GoogleCloudDataproc/hadoop-connectors/blob/master/gcs/README.md) can be used for reading files from a GCS bucket from a Spark application running on Ray on Vertex AI. This is done using a few configuration parameters when a Spark session is created using RayDP. The following code shows how a CSV file stored in a GCS bucket can be read from a Spark application.\n",
```

Comment:
remove abbreviation "GCS"and use "Google Storage",  and add a link 
    - "reading files from a [Cloud Storage bucket](https://cloud.google.com/storage/docs/buckets) from a Spark"